tests/test_torch_compile.py

```diff
@@ -1,11 +1,14 @@
 import os
+import platform
+import packaging
 
 import gym
 import torch as th
 
 from stable_baselines3 import SAC
 
-
+@pytest.mark.skipif(platform.system() == "Windows", reason="Windows does not support torch.compile.")
+@pytest.mark.skipif(packaging.version.parse(th.__version__.split) < "2.0.0", reason="PyTorch version does not support torch.compile.")
 def test_load_compiled():

```

Comment:
I forgot: please use a tmp path (generated by pytest), same remark as in https://github.com/DLR-RM/stable-baselines3/pull/1463#discussion_r1176603209